src/OrchardCore/OrchardCore.Localization.Core/DataAnnotations/DataAnnotationsDefaultErrorMessages.cs

```diff
@@ -0,0 +1,118 @@
+using Microsoft.Extensions.Localization;
+
+namespace OrchardCore.Localization.Core.DataAnnotations
+{
+    internal class DataAnnotationsDefaultErrorMessages
+    {
+        private readonly IStringLocalizer<DataAnnotationsDefaultErrorMessages> S;
+
+        public DataAnnotationsDefaultErrorMessages(IStringLocalizer<DataAnnotationsDefaultErrorMessages> localizer)
+        {
+            S = localizer;
+        }
+
```

Comment:
Is this class actually used? Or is it just a marker class that the POExtractor will use with reflection to find and add the strings?

If it is just a marker, does it need the constructor? 
Or does the PO extractor instantiate an instance of it?

And could it have a comment explaining that it is a marker, and is not for use?

Reply:
Ya it's sort of marker, that's why I declare it as `internal`, perhaps you are right no need for the constructor, but it may be useful in some use cases when you need to get a specific localized error message

+1 for adding a comment, so let me create a PR and ping you there

Thanks